akka-http-bench-jmh/src/main/scala/akka/http/impl/engine/http2/H2ServerProcessingBenchmark.scala

```diff
@@ -71,6 +71,7 @@ class H2ServerProcessingBenchmark extends CommonBenchmark {
            #akka.loglevel = debug
            akka.actor.default-dispatcher.fork-join-executor.parallelism-max = 1
            #akka.http.server.log-unencrypted-network-bytes = 100
+           akka.http.server.http2.max-concurrent-streams = 50000 # needs to be higher than the above `numRequests`
```

Comment:
I guess we could use something like `${numRequests * 5}` to make this always true?

Reply:
I think even `numRequests` is enough.

Reply:
Done